protos/failure/failure.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package mavsdk.rpc.failure;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.failure";
+option java_outer_classname = "TuneProto";
+
+// Inject failures into system to test failsafes.
+service FailureService {
+  // Injects a failure.
+  rpc Inject(InjectRequest) returns(InjectResponse) { option (mavsdk.options.async_type) = SYNC; }
+}
+
+// A failure unit.
+enum FailureUnit {
+    FAILURE_UNIT_SENSOR_GYRO = 0; // Gyro
+    FAILURE_UNIT_SENSOR_ACCEL = 1; // Accelerometer
+    FAILURE_UNIT_SENSOR_MAG = 2; // Magnetometer
+    FAILURE_UNIT_SENSOR_BARO = 3; // Barometer
+    FAILURE_UNIT_SENSOR_GPS = 4; // GPS
+    FAILURE_UNIT_SENSOR_OPTICAL_FLOW = 5; // Optical flow
+    FAILURE_UNIT_SENSOR_VIO = 6; // Visual inertial odometry
+    FAILURE_UNIT_SENSOR_DISTANCE_SENSOR = 7; // Distance sensor
+    FAILURE_UNIT_SENSOR_AIRSPEED = 8; // Airspeed
+    FAILURE_UNIT_SYSTEM_BATTERY = 100; // Battery
```

Comment:
I know, I was torn to on how to do that. It's probably best to not have them match.

Reply:
If they don't match it means that I need to create a function just matching enums which is not fun.
If they do match, I can just use `static_cast` which is so convenient. I'm being lazy here as I'm usually the one pushing back on exactly this. :smile:

Reply:
> I can just use static_cast which is so convenient.

As you want, but we will have to be careful when this gets extended, because it may become a bug then :slightly_smiling_face: